src/files/list-thumbnail.js

```diff
@@ -127,13 +128,7 @@ class RawListThumbnailFile extends BaseFile {
   }
 }
 
-@DragSource('file', BaseFileConnectors.dragSource, BaseFileConnectors.dragCollect)
-@DropTarget(
-  ['file', 'folder', NativeTypes.FILE],
-  BaseFileConnectors.targetSource,
-  BaseFileConnectors.targetCollect
-)
 class ListThumbnailFile extends RawListThumbnailFile {}
 
-export default ListThumbnailFile
+export default flow(DragSource('file', BaseFileConnectors.dragSource, BaseFileConnectors.dragCollect), DropTarget(['file', 'folder', NativeTypes.FILE], BaseFileConnectors.targetSource, BaseFileConnectors.targetCollect))(ListThumbnailFile)
```

Comment:
It's a shame that utilising hooks would require a large refactor. Don't like how this reads. Some whitespace might help.

`ListThumbnailFile` doesn't need to exist anymore

```suggestion
export default flow(
  DragSource('file', BaseFileConnectors.dragSource, BaseFileConnectors.dragCollect),
  DropTarget(['file', 'folder', NativeTypes.FILE], BaseFileConnectors.targetSource, BaseFileConnectors.targetCollect)
)(RawListThumbnailFile)
```

Reply:
You're right. It does look messy. I'll take care of it :)

As for the `ListThumbnailFile` I only kept the naming conventions for the sake of helping understand how the code translated from decorators -> HoCs. If not needed, I will simplify further for each occasion it is used at